async_upnp_client/profiles/profile.py

```diff
@@ -29,6 +29,15 @@
 RESUBSCRIBE_TOLERANCE_SECS = RESUBSCRIBE_TOLERANCE.total_seconds()
 
 
+def _find_device_of_type(device: UpnpDevice, device_types: List[str]) -> UpnpDevice:
```

Comment:
Can this be exposed as a part of the `UpnpProfileDevice` API, to verify that a given `UpnpDevice` is usable for a `UpnpProfileDevice` subclass without first constructing it?